user/enterprise/build-images.md

```diff
@@ -75,85 +75,31 @@ To build docker images on Travis CI Enterprise, add the following to `/etc/defau
-
-```json
-{
-    //...
-    "hosts": [
-        "fd://",
-        "tcp://172.31.7.199:4243"
-    ]
-}
-```
-
-By default, the Docker daemon is configured to be accessible by  [socket activation](http://0pointer.de/blog/projects/socket-activation.html) (that's what `fd://` is for). To prevent issues related to having host configured in both service file and `/etc/docker/daemon.json` edit the Docker settings by by running `sudo systemctl edit docker`. It opens an editor with a new file where you can override Docker default settings. Please add the following in there:
-
-```
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd
-```
-
-You'll notice that in the above code snippet `ExecStart` gets assigned twice: first time with an empty value and the second time with a new command. This is a specific systemd semantic we need to follow here in order to customize the Docker start command.
-
-After that, issue an `sudo systemctl daemon-reload`, together with a `sudo systemctl restart docker`.
+With this option we tell `travis-worker` to make the host's Docker socket available inside the build containers. Please restart travis-worker after you have saved the configuration file.
 
 
 __Restart travis-worker__
 
-To restart travis-worker, you can find the instructions [here](https://docs.travis-ci.com/user/enterprise/worker-cli-commands/#Stopping-and-Starting-the-Worker).
-
+To restart travis-worker, you can find the instructions [here](/user/enterprise/worker-cli-commands/#Stopping-and-Starting-the-Worker).
 
 ### Updates to your .travis.yml files
 
 #### Trusty build containers
 
-> Please note: This solution utilizes the Docker daemon from the host machine - any containers and images you pull and run through your build need to be cleaned up manually.
-
-Add the following to your `.travis.yml` configuration:
-
-```
-install:
-  - sudo apt-get update
-  - sudo apt-get install -y curl software-properties-common apt-transport-https ca-certificates
-  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu trusty stable"
-  - sudo apt-get update
-  - sudo apt-get install -y docker-ce
-  - sudo docker -H <IP>:4243 pull ubuntu
-script:
-  - sudo docker -H <IP>:4243 run ubuntu date
-
-```
+Once the worker machine is [configured properly](/user/enterprise/build-images/#worker-machine-configuration), you can use Docker as usual in your build. Please note that on an Enterprise installation you don't need to add `services: docker` to the `.travis.yml`.
+Since you're using the host's Docker daemon, all images and containers participating in your build stay on the machine. Because of that, we recommend to use the `--rm` flag when you use `docker run`. To avoid race conditions when it comes to cleaning up build images during the build, we recommend to clean them up manually on the machine directly while no build is running.
```

Comment:
This last sentence feels a bit muddled. Let's split it up a bit:

> Since you're using the host's Docker daemon, all images and containers used in your build are stored on the host machine. To free up disk space, we recommend using the  `--rm` flag when you use `docker run` in your build.

Then I don't really understand the connection with:

> To avoid race conditions when it comes to cleaning up build images during the build, we recommend cleaning them up manually on the host machine directly while no build is running.